src/core/RMXRemixer.m

```diff
@@ -141,22 +130,6 @@ - (void)didSwipe:(UISwipeGestureRecognizer *)recognizer {
   [_overlayController showPanelAnimated:YES];
 }
 
-#pragma mark - Email
-
-+ (void)sendEmailInvite {
```

Comment:
I know this was hard-coded here... but will we still be able to provide some mechanism to get the generated device ID and firebase website to go to?